server/bin/insert-test-data-fast.sh

```diff
@@ -12,7 +12,7 @@ CONF_DIR="${BASE_DIR}/conf"
 CONF_FILE="${CONF_DIR}/timely.properties"
 LIB_DIR="${BASE_DIR}/lib"
 NUM_SERVER_THREADS=4
-HOST="127.0.0.1"
+HOST="timely"
```

Comment:
I think the better option is to specify the host/port on the command line. If nothing is specified, they can be changed to the previous defaults.